demo/cmd/common/books.go

```diff
@@ -19,7 +19,7 @@ const (
 	RestockWarehouseURL = "restock-books"
 
 	// bookstorePort is the bookstore service's port
-	bookstorePort = 80
+	bookstorePort = 8080
```

Comment:
I know this is a work in progress, but we shouldn't be changing to default demo ports to 8080 just for this. If for some reason this port is unusable for Openshift, we should simply override the port similar to other configurable options.

In the past we did change the demo ports to 8080 and back to 80, more details can be found here https://github.com/openservicemesh/osm/issues/1099#issuecomment-660197168

Reply:
Updated in favor of using a different port